### DISCUSSION TITLE
Is the Prisma ORM Quickstart guide meant for a Mac?

### DISCUSSION THREAD

**Discussion by givihuda** [prisma/prisma#22928] [Q&A]
### Question

Just curious because its first command instructions are for Linux or Unix-like CLI:

```
mkdir hello-prisma 
cd hello-prisma
```

https://www.prisma.io/docs/getting-started/quickstart

I tend to use Windows for development, but I'm not a heavy user of JavaScript or its related libraries and frameworks, I'm just checking Prisma ORM out since I recently heard about it. I've used Linux a lot but I don't currently have that or a Mac.

**Comment by ludralph** [comment]
Hi @givihuda 👋 

Thank you for raising this question.

The commands you've mentioned `mkdir hello-prisma and cd hello-prisma` are used to create a new directory and navigate into it. These commands are not specific to Linux or Unix-like CLI, they can also be used in the Windows Command Prompt.
To set up Prisma ORM on your Windows machine, you can follow these steps:
Navigate into your project directory that contains the package.json file.
Add the Prisma CLI as a development dependency to your project using the following command:
```
npm install prisma --save-dev
```
You can now invoke the Prisma CLI by prefixing it with npx. Set up your Prisma project by creating your Prisma schema file template with the following command:
```
npx prisma init
```
These instructions are applicable regardless of the operating system you are using. For more detailed instructions, you can refer to the [documentation](https://www.prisma.io/docs/getting-started/setup-prisma).

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
